homeassistant/components/august/lock.py

```diff
@@ -44,7 +44,7 @@ class AugustLock(AugustEntityMixin, RestoreEntity, LockEntity):
     def __init__(self, data: AugustData, device: Lock) -> None:
         """Initialize the lock."""
         super().__init__(data, device)
-        self._lock_status = None
+        self._lock_status: LockStatus | None = None
```

Comment:
Could be set outside of the constructor

Reply:
2632a13f1df1230591e2ca82d62d5bc133cabe58